sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -253,26 +255,30 @@ public partial class ContainerRegistryBlobClient
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Containers.ContainerRegistry.Specialized.UploadManifestResult>> UploadManifestAsync(Azure.Containers.ContainerRegistry.Specialized.OciManifest manifest, Azure.Containers.ContainerRegistry.Specialized.UploadManifestOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Containers.ContainerRegistry.Specialized.UploadManifestResult>> UploadManifestAsync(System.IO.Stream manifestStream, Azure.Containers.ContainerRegistry.Specialized.UploadManifestOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
-    public partial class DownloadBlobResult : System.IDisposable
+    public partial class DownloadBlobResult
     {
         internal DownloadBlobResult() { }
-        public System.IO.Stream Content { get { throw null; } }
+        public System.BinaryData Content { get { throw null; } }
         public string Digest { get { throw null; } }
-        public void Dispose() { }
+    }
+    public partial class DownloadBlobToOptions
+    {
+        public DownloadBlobToOptions(int maxChunkSize) { }
```

Comment:
We use the [default chunk size](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/ContainerRegistryBlobClient.cs#L22) if options isn't passed ... options itself is optional.  If we don't ask the user for the blob size, this would be the only option someone could pass, so I think it makes sense to have a constructor that requires it ... the .NET pattern for options types wants us to set required values in the constructor.

Reply:
Hm, I like the idea to move chunk size to a client-level option.  Let me look into this - thanks!

Reply:
Added to ClientOptions.